virttest/libvirt_xml/vm_xml.py

```diff
@@ -838,9 +848,18 @@ def set_vm_vcpus(vm_name, vcpus, current=None, sockets=None, cores=None,
                                     nodexml["cpus"] = str(index + vcpu_rem)
                                 else:
                                     nodexml["cpus"] = str(index)
+                            if numa_number > 0:
+                                nodexml['id'] = str(node)
+                                nodexml['memory'] = str(vmxml.max_mem // numa_number)
                             index = vcpus_num * (node + 1)
                             nodexml_list.append(nodexml)
-                        vmcpu_xml.set_numa_cell(nodexml_list)
+                            logging.debug(nodexml)
+                        logging.debug(nodexml_list)
```

Comment:
Would you please provide more proper messages?

Reply:
I will delete it. I just use it to debug.